editoast/Cargo.toml

```
@@ -15,11 +15,13 @@ clap = { version = "4.1", features = ["derive", "env"] }
 colored = "2.0.0"
 chrono = { version = "0.4", features = ["serde"] }
 derivative = "2.2.0"
+uuid = { version = "1.3.0", features = ["v4"] }
```

To use the same version as the one used by diesel, we should only lock the major version. Checkout https://github.com/diesel-rs/diesel/blob/9fea8dfb98389acde9bd5c1666131da1c8c8ce24/diesel/Cargo.toml#L26

```suggestion
uuid = { version = "1", features = ["v4"] }
```